platform-ui-skin/src/main/webapp/skin/less/vuetify/lib/README.txt

```diff
@@ -17,3 +17,4 @@ Modifications applied on imported libraries:
   - .v-tabs--centered > .v-tabs-bar .v-tabs-slider-wrapper (centered tabs) doesn't use centered tabs because of bad usage of CSS priorities, thus !important is added to effectively center tabs
   - delete '!important' from .v-card-flat:@box-shadow (line 22363) to avoid force disabling box-shadow on cards when using flat attribute on v-card
   - Add fix on v-alert.prominent.dense icon size introduced in 2.4
+  - Add @borderRadius with 5px instead of using 4px for defaut border radius
```

Comment:
```suggestion
  - Add @borderRadius with 4px instead of using 5px for defaut border radius
```

Reply:
```suggestion
  - Add @baseBorderRadius variable to use centralized border radius for Vuetify components
```

@SaraBoutej we may need to delete the usage of `@borderRadius` here. In fact, the brandable `@borderRadius` should be used for applications parent DOM element only.

Reply:
fixed